docs/usage/usage-with-typescript.md

```diff
@@ -9,13 +9,44 @@ hide_title: true
 
 Redux Toolkit is written in TypeScript, and its API is designed to enable great integration with TypeScript applications.
 
-This page is intended to give an overview of all common usecases and the most probable pitfalls you might encounter when using RTK with TypeScript.
+The first part of this page is intended to give you an overview over how to do a basic setup that every application requires. The second part goes into the specifics of the different APIs.
 
 **If you encounter any problems with the types that are not described on this page, please open an issue for discussion.**
 
+## Basic Project Setup
+
+Using [configureStore](../api/configureStore.mdx) should not need any additional typings. You will, however, want to extract the `RootState` type and the `Dispatch` type.
+Also, you will want to create typed versions of the `useDispatch` and `useSelector` hooks for usage in your application.
+
+That said, a basic setup would look like this:
+
+```ts title="store.ts"
```

Comment:
split up into two files

Reply:
Not seeing "two files" here - just the one "Usage with TS" page.  Did you forget to push something, or am I missing what you're referring to?

Reply:
Oh wait, you mean this _example_ is in two files now. got it.